Frontend/src/components/TaskList.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it, vi, Mocked } from 'vitest';
+import TestRenderer from 'react-test-renderer';
+import { render, screen } from '@testing-library/react';
+import axios from 'axios';
+import TaskList, { getTasks } from './TaskList';
+
+vi.mock('axios');
+
+describe('TaskList.tsx', () => {
```

Comment:
You can simply write `TaskList` in the describe block, no need for `.tsx` :)

Reply:
Oops. That was silly.